src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -1094,44 +1094,46 @@ public function providePathsForMakePathRelative()
         $paths = array(
             array('/var/lib/symfony/src/Symfony/', '/var/lib/symfony/src/Symfony/Component', '../'),
             array('/var/lib/symfony/src/Symfony/', '/var/lib/symfony/src/Symfony/Component/', '../'),
-            array('/var/lib/symfony/src/Symfony', '/var/lib/symfony/src/Symfony/Component', '../'),
-            array('/var/lib/symfony/src/Symfony', '/var/lib/symfony/src/Symfony/Component/', '../'),
+            array('/var/lib/symfony/src/Symfony/', '/var/lib/symfony/src/Symfony/Component', '../'),
```

Comment:
not sure why you change some tests...? Can you give more context? 🤔

Reply:
seems like he just removed a duplicated assertion :)

Reply:
I've remove some dupplicate and the third point of the issue says that we should be able to dectect the trailing slash, that's why.

Reply:
I agree. But we should not remove the tests without the trailing slash either, just add new ones as both ways should work :-)